src/omero/plugins/duplicate.py

```diff
@@ -61,6 +72,38 @@ def cmd_type(self):
         import omero.all
         return omero.cmd.Duplicate
 
+    def _pre_objects(self, parser):
+        parser.add_argument(
+            "--duplicate-classes",
+            help=("Modifies the given option by specifying kinds of object to "
+                  "duplicate"))
+        parser.add_argument(
+            "--reference-classes",
+            help=("Modifies the given option by specifying kinds of object to "
+                  "link to instead of duplicate"))
+        parser.add_argument(
+            "--ignore-classes",
```

Comment:
These could conceivable be marked `nargs="+"` to handle `-R Foo -R Bar`.

Reply:
That one's new to me so have created https://github.com/ome/omero-py/issues/253 as a start.